### DISCUSSION TITLE
Add postgis types support and allow custom types

### DISCUSSION THREAD

**Discussion by insanebaba** [SeaQL/sea-orm#651] [General]
Hi team, 
I want to use postgis types, but I'm not sure how to do it.

I can use geotypes crate but it's not allowed in entity, 

Can you please support geotypes and also allow to add custom types?

**Comment by billy1624** [comment]
Hey @insanebaba, sorry for the late reply. It's currently not supported. Seems that it's not supported in SQLx at this stage too.
- launchbadge/sqlx/issues/166

**Comment by billy1624** [comment]
Related #336

**Comment by tumluliu** [comment]
Is there any further activity planned on this issue? Is it still being blocked by https://github.com/launchbadge/sqlx/issues/166 ? I would like to contribute as it's a big pity to not have a [GeoAlchemy2](https://github.com/geoalchemy/geoalchemy2) equivalent from rust/SeaORM world. Sth. like geoSeaORM

**Comment by pentatonick** [comment]
~~Linked issue https://github.com/launchbadge/sqlx/issues/166 was Closed as completed 2 week ago~~

**Reply by netthier** [reply]
The original question was about PostGIS types (so geometry), not the PostgreSQL-native geometric type

**Reply by pentatonick** [reply]
Thanks for pointing it out, I stand corrected.
